### DISCUSSION TITLE
I'm not sure but, is there a Windows Desktop Version availabel of the QZ App?

### DISCUSSION THREAD

**Discussion by quaker75** [cagnulein/qdomyos-zwift#2575] [General]
As I can read in some discussions, Windows is always mentioned. If there is a "standalone" version for windows available, this would be great because I will use only a Windows desktop for Zwift and my Garmin Tacx. I need QZ only for virtual shifting in Zwift. Is there a possibility to do this without a smartphone, directly with the Windows PC?

**Comment by cagnulein** [comment]
yes you can download it from here! https://github.com/cagnulein/qdomyos-zwift/releases/download/nightly/windows-msvc2019-binary-no-python.zip

Let me know if you have any other questions!

**Reply by quaker75** [reply]
![qz-windows](https://github.com/user-attachments/assets/c2962966-44f9-44a7-96a7-a9df60032034)
Hi again,
are there any known problems with windows? Its always "not responding" on my system
Best regards
Sven

**Reply by cagnulein** [reply]
Did you try to wait a little? Does it hangs also on the main screen?

**Comment by quaker75** [comment]
yes, I wait for at least  10 Minutes, still hanging.  just closing works.
Hanging on the mainscreen (i'm not sure) but i think not. It just hangs after i do some clicks in the app.

**Comment by cagnulein** [comment]
are you able to create a debug log?
https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

Windows 10 or 11? Which Bluetooth dongle?

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 6 set 2024 alle 13:25 quaker75 ***@***.***> ha
scritto:

**Comment by quaker75** [comment]
Hi,
its Windows 11.
BT Dongle is this one:
https://www.amazon.de/gp/product/B0CZD94YFR/ref=ppx_yo_dt_b_search_asin_title?ie=UTF8&psc=1

[debug-Fr_Sep_6_13_59_58_2024.log](https://github.com/user-attachments/files/16907453/debug-Fr_Sep_6_13_59_58_2024.log)
![qt-windows-not-responding](https://github.com/user-attachments/assets/34002db4-16fc-46a1-8c7c-b997ee0d431f)
Regards
Sven

**Comment by cagnulein** [comment]
i guess the issue could be on windows 11. do you have a windows 10 pc just
to check that this is the issue? or a even a virtual machine.
let me know

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 6 set 2024 alle ore 14:06 quaker75 ***@***.***>
ha scritto:

**Comment by quaker75** [comment]
i can check on a win10 vm, but without any bt components. if this is enough?

**Reply by cagnulein** [reply]
no of course it needs a bluetooth dongle. you can add the bt device into the VM for example in virtualbox

**Comment by quaker75** [comment]
ok, I've tested it on a windows 10 notebook with integrated bluetooth. sometimes it hangs, but only for a very short time, and then the programm works again. any id how to fix on my win 11?

**Reply by cagnulein** [reply]
i will try to setup a windows 11 machine. in the meantime you can use this VM

**Comment by cagnulein** [comment]
i guess i can replicate this on windows 11. let me see if i can fix it

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno ven 6 set 2024 alle ore 14:27 quaker75 ***@***.***>
ha scritto:

**Comment by cagnulein** [comment]
@quaker75 by they way, do you have the https://aka.ms/vs/17/release/vc_redist.x64.exe installed on windows 11?

**Comment by cagnulein** [comment]
ok i guess i fixed it, i'm building a new one for you @quaker75

**Comment by cagnulein** [comment]
@quaker75 done, it's building here https://github.com/cagnulein/qdomyos-zwift/actions/runs/10740060131
at the end, you will see the download link at the bottom of the page!
Let me know!

**Reply by quaker75** [reply]
you're very fast :) 
But me also!
In the meantime I downgraded to Win 10.
So I will test the new Version with Win 11 again, after an new upgrade of my system
Many thanks!

**Reply by cagnulein** [reply]
@quaker75 Did you ever test it again on win11?

**Comment by quaker75** [comment]
Hi, Not yet
Regards Sven

Am 28.10.2024 um 20:24 schrieb Roberto Viola ***@***.***>:

﻿

@quaker75<https://github.com/quaker75> Did you ever test it again on win11?

**Comment by oantoshkiv** [comment]
Windows app exiting silently after couple of minutes when activity started.
Didn't notice errors in debug log, but there is zero sized backup fit file created and some other zero sized file with random 5 characters in name. Zwift is running on same PC. Windows 10.
R1 Pro treadmill.
I tried Android app, it works, Zwift can find Wahoo TREAD 0003, but treadmill speed in Zwift is always 0.

**Reply by cagnulein** [reply]
@oantoshkiv Are you running qz as administrator?

For android, on which device zwift runs? Did you try to run on the same android enabling the android notification icon setting in the experimental settings and restart qz?
